samples/drivers/watchdog/boards/gd32e103v_eval.conf

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2022 TOKITA Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_GD32_DBG_SUPPORT=y
```

Comment:
Since we relaxed this condition condition, you can drop all this *.conf

Reply:
Remove all *.conf